src/api/types.ts

```diff
@@ -233,3 +234,57 @@ export interface OpenmrsFormResource extends OpenmrsResource {
   dataType: string;
   valueReference: string;
 }
+
+export interface ProviderResponse {
+  results: Array<Provider>;
+}
+
+export interface Provider {
+  uuid: string;
+  display: string;
+  person: {
+    uuid: string;
+    display: string;
+  };
+  identifier: string;
+  attributes: [];
+  retired: false;
+}
+
+export interface LocationResponse {
+  results: Array<Location>;
+}
+
+export interface Location {
+  uuid: string;
+  display: string;
+  name: string;
+  description: string;
+  address1: string;
+  address2: string;
+  cityVillage: string;
+  stateProvince: string;
+  country: string;
+  postalCode: string;
+  latitude: string;
+  longitude: string;
+  countyDistrict: string;
+  tags: [
+    {
+      uuid: string;
+      display: string;
+    },
+  ];
+  parentLocation: {
+    uuid: string;
+    display: string;
+  };
+  childLocations: [
+    {
+      uuid: string;
+      display: string;
+    },
+  ];
+  retired: boolean;
+  attributes: [];
```

Comment:
Not sure if we need all these attributes, if not then i can make adjustments to the endpoint only to return the necessary fields